theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -904,6 +908,9 @@ def tcase(self, i, f, s, b, flip, provide_shape, fd=(1, 1)):
                           filter_dilation=fd)
 
     def tcase_gi(self, i, f, o, s, b, flip, provide_shape, fd=(1, 1), expect_error=False):
+        if not theano.config.cxx:
+            raise SkipTest("Need cxx to test conv2d")
```

Comment:
This one as well, as Conv2d has Python code.
We may decide it would be too long to execute, but it would at least deserve a comment, and it was tested before.